### DISCUSSION TITLE
Suggestion: Reposition the Remnant systems

### DISCUSSION THREAD

**Discussion by MidnightPlugins** [endless-sky/endless-sky#10313] [Ideas]
_“It’s not about the destination, it’s about the journey.”_

Reposition the Remnant systems so that the uninhabited systems sit between the inhabited ones. Having intermediary systems creates story opportunities for stuff to happen while traveling from one Remnant system to another.

Example:

- the positions of Cinxia and Perfica are swapped
- the link between Arculus and Pantica is replaced with a link between Farinus and Pantica

![remnant repositioned systems](https://github.com/user-attachments/assets/102813e4-7279-4b6f-ad48-fee28bc0de9c)

Example modified 0.10.8 map file: 
[map systems.txt](https://github.com/user-attachments/files/16188692/map.systems.txt)

**Comment by TheGiraffe3** [comment]
There is that one rush storyline mission where you need to go pick someone up and you have a very short timeline. That would have to be changed, and potentially other plans for the Remnant storyline that @Zitchas is working on.

**Comment by Azure3141** [comment]
Honestly I've always thought it was a little odd that all the Remnant planets were clustered together in a neat little triangle instead of being scattered around further.

**Comment by lumbar527** [comment]
I kind of like the neat, organized Remnant systems. It's convenient to travel through. The map change, just looking at it, makes them feel bigger.
